exporters/otlp/internal/otlp_exporter_test.cc

```diff
@@ -1,4 +1,4 @@
-#include "otlp_exporter.h"
+#include "exporters/otlp/otlp_exporter.h"
```

Comment:
Ideally, we'd be able to include via `opentelemetry/exporters/otlp/otlp_exporter.h`, but I think this wouldn't be possible without maintaining custom `include` directories for all exporters.

I'm fine with leaving out the `opentelemetry` include prefix for internal exporters, but let's see what other approvers say.

Reply:
I don't have strong opinion here, but I slightly prefer specifying full path starting from "opentelemetry" whenever it is possible for consistency.

Reply:
Do we expect the exporter headers to be included anywhere other than the exporter itself? If not, it may be overkill to maintain an `include` directory for each exporter. Please let me know your thoughts!

Reply:
I've added an `include` directory for the headers. Please let me know which version you prefer (with or without the `include` directory). Personally I think having the `include` directory over-complicates things a bit, but of course I  understand the value of consistency as well. It might also make it easier to extend the project in the future.

Reply:
I like the `include` directory approach. It might seem like a bit of an overkill, but it gives us more flexibility, makes for clear and controlled include paths and is consistent with the [directory structure elsewhere](https://github.com/open-telemetry/opentelemetry-cpp/tree/master/sdk). Thanks for coming up with this approach.

The CMake build should work if you add a `include_directories(include)` to the `CMakeLists.txt` of the OTLP exporter.

Reply:
I like the `include` directory approach. It might seem like a bit of an overkill, but it gives us more flexibility, makes for clear and controlled include paths and is consistent with the [directory structure elsewhere](https://github.com/open-telemetry/opentelemetry-cpp/tree/master/sdk). Thanks for coming up with this approach.

The CMake build should work if you add a `include_directories(include)` to the `CMakeLists.txt` of the OTLP exporter.

Reply:
Sounds good, thank you!